### DISCUSSION TITLE
thirty/%E5%9C%A8Linux%E4%B8%8A%E9%83%A8%E7%BD%B2%E5%90%84%E7%B1%BB%E8%BD%AF%E4%BB%B6

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#653] [Announcements]
# thirty/%E5%9C%A8Linux%E4%B8%8A%E9%83%A8%E7%BD%B2%E5%90%84%E7%B1%BB%E8%BD%AF%E4%BB%B6

MySQL 5.7版本安装 注意：安装操作需要root权限 配置yum仓库 更新密钥 rpm --import https://repo.mysql.com/RPM-GPG-KEY-mysql-2022 安装Mysql yum库 rpm -Uvh http://repo.mysql.com//mysql57-community-release-el7-...

https://machaojin.cn/thirty/%E5%9C%A8Linux%E4%B8%8A%E9%83%A8%E7%BD%B2%E5%90%84%E7%B1%BB%E8%BD%AF%E4%BB%B6.html

**Comment by skxndbzu** [comment]
太对列
